src/logic/operate.js

```diff
@@ -0,0 +1,16 @@
+export default function operate(currentInput, operation) {
+  const currentNum = parseFloat(currentInput);
+
+  if (Number.isNaN(currentNum)) {
+    return 'Error';
+  }
+
+  switch (operation) {
+    case '+/-':
+      return (-currentNum).toString();
```

Comment:
- [ ] Just as I pointed out above, you are not meant to modify the content of the two logic files. Microverse already tested the functionality, I'll suggest you go through your project requriwmnt one more time and understand how to use these two file. I'll also suggest you [visit this link and get the original ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js)provided file. :smile: :+1: